### DISCUSSION TITLE
2022.11.12

### DISCUSSION THREAD

**Discussion by itmakesmesoft** [itmakesmesoft/Steady-Study#152] [Self-Assessment]
이번 주 목표를 이루기 위해 어떤 노력을 했는지 스스로 평가해보고, 점수를 매겨주세요.
ex)
이번 주 목표 : 기술 블로그에 배운 내용 정리하기

점수 : 80점
이유 : (생략)

**Comment by itmakesmesoft** [comment]
이은혁 : 자바 공부 시작하고, 기록하기
고영석 : 강의 따라가기, 알고리즘 2개 풀고 블로그에 올리기
나혜승 : 알고리즘 두문제 풀기, viEW 실습 커피 주문 어플 혼자서
이수연 : 1일 1알고리즘, 자바 기초문법 끝내기, Django로 간단한 웹페이지 만들기(1/2)
이문삼 : 강의 따라가기, 리액트랑 VIEW 블로그 포스팅 5개하기

**Comment by itmakesmesoft** [comment]
이번 주 목표 : 자바 공부 시작하고, 기록하기

점수 : -100점
이유 : 
목표를 늦게 올림 => -100점
자바 공부 시작함 => +10점
기록 안함 => -10점

**Comment by moonthree** [comment]
이번주 목표 : 강의 따라가기, 리액트랑 VIEW 블로그 포스팅 5개하기

점수 : 40점
이유 :
강의만 따라가고 블로그 포스팅은 전혀 못 함

**Comment by sooyeonlee127** [comment]
이번 주 목표 : 1일 1알고리즘, 자바 기초문법 끝내기, Django로 간단한 웹페이지 만들기(1/2)
점수: 30점
이유:
1일 1알고리즘(성공), 자바 기초문법(상속까지만.. 했습니다..ㅠ), Django로 간단한 웹페이지 만들기(전혀 못함)

**Comment by HyeseungNA** [comment]
이번 주 목표 : 알고리즘 두문제 풀기, VIEW 실습 커피 주문 어플 혼자서 만들기

점수 : 50점

이유: 알고리즘은 한 문제밖에 못 풀었고, 커피주문 어플은 다 만들지는 못했지만 매일 기능 하나씩 추가하려고 노력했습니다

**Comment by YoungSeok222222** [comment]
이번주 목표: 강의 따라가기, 알고리즘 2개 풀고 블로그에 올리기

점수: 25점
이유:  먼저 알고리즘 2개 풀고 블로그에 올리는 것을 못했습니다. 강의는 절반만 따라간거 같기 때문에 점수를 반의 반인 25점을 줬습니다.

**Comment by ats166** [comment]
이번 주 목표 : 강의 따라가기, 1일 1알고리즘, 블로그 포스팅
점수 : 30점 
이유 : 회식한다고 알고리즘을 몇번 빼먹었으며 강의도 수업중 프로젝트를 제대로 완수하지못한경우가 많아서 못따라갔다고생각합니다.

**Comment by Chung-HyeSu** [comment]
이번주 목표 : 관통프로젝트 준비하기(vue, django복습 등), 시간 밀도있게 쓰기,
